### PR TITLE
ci: Update branches to master

### PR DESCRIPTION
**Pull request by francislavoie**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -123,4 +123,4 @@ jobs:
         uses: reviewdog/action-golangci-lint@v1
         # uses: docker://reviewdog/action-golangci-lint:v1 # pre-build docker image
         with:
-          github_token: ${{ secrets.github_token }}
\ No newline at end of file
+          github_token: ${{ secrets.github_token }}
```
Comment (francislavoie) [inline .github/workflows/ci.yml]: Did the commit in GitHub, it added a newline to the file on its own I guess. Whatever.

**Comment by francislavoie** [comment]
Crazy... windows tests failed for some reason.

```
--- FAIL: TestRedirect (0.47s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x0 pc=0x7753f5]

goroutine 78 [running]:
testing.tRunner.func1.1(0x1e33ae0, 0x4494630)
	C:/hostedtoolcache/windows/go/1.14.1/x64/src/testing/testing.go:941 +0x5d7
testing.tRunner.func1(0xc000731e60)
	C:/hostedtoolcache/windows/go/1.14.1/x64/src/testing/testing.go:944 +0x607
panic(0x1e33ae0, 0x4494630)
	C:/hostedtoolcache/windows/go/1.14.1/x64/src/runtime/panic.go:973 +0x3f1
net/url.(*URL).String(0x0, 0x0, 0x36b4fe0)
	C:/hostedtoolcache/windows/go/1.14.1/x64/src/net/url/url.go:777 +0x65
github.com/caddyserver/caddy/v2/caddytest.AssertRedirect(0xc000731e60, 0x2050b7c, 0x16, 0x205e62c, 0x1b, 0x12d, 0x0)
	D:/a/caddy/caddy/caddytest/caddytest.go:304 +0x241
github.com/caddyserver/caddy/v2/caddytest/integration.TestRedirect(0xc000731e60)
	D:/a/caddy/caddy/caddytest/integration/caddyfile_test.go:49 +0xaf
testing.tRunner(0xc000731e60, 0x20dcff0)
	C:/hostedtoolcache/windows/go/1.14.1/x64/src/testing/testing.go:992 +0x1f3
created by testing.(*T).Run
	C:/hostedtoolcache/windows/go/1.14.1/x64/src/testing/testing.go:1043 +0x668
FAIL	github.com/caddyserver/caddy/v2/caddytest/integration	4.132s
```

**Comment by francislavoie** [comment]
It makes no sense though, cause it immediately passed just before, in the previous commit. There should have been absolutely no difference there.

**Comment by francislavoie** [comment]
K, it passed on the re-run. Whatever. :man_shrugging: 

Does that stack trace tell you anything though?

**Comment by mholt** [comment]
This whole thing tells me there's a race in the integration tests -- or _something_ that's nondeterministic, which needs fixing. 🤓 

Will look into it soon, if @sarge doesn't beat me to it! He knows that code better.

**Pull request merged**